modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/DropdownMenu.js

```diff
@@ -49,16 +50,16 @@ export default function DropdownMenu({
 				})}
 				trigger={
 					<ClayButton className={cssClass} {...otherProps}>
-						{icon && (
-							<span
-								className={classNames({
-									'inline-item': label,
-									'inline-item-before': label,
-								})}
-							>
+						{icon &&
+							(label ? (
+								<span
+									className={'inline-item inline-item-before'}
```

Comment:
We don't need to use `{'...'}` here, it's just a string.

Reply:
Oh! You're right! :) Thanks!